src/imitation/scripts/train_imitation.py

```diff
@@ -140,6 +141,41 @@ def dagger(
     return stats
 
 
+@train_imitation_ex.command
+def sqil(
+    sqil: Mapping[str, Any],
+    _run,
+    _rnd: np.random.Generator,
+) -> Mapping[str, Mapping[str, float]]:
+    custom_logger, log_dir = logging_ingredient.setup_logging()
+    expert_trajs = demonstrations.get_expert_trajectories()
+
+    with environment.make_venv() as venv:
+        sqil_trainer = SQIL(
+            venv=venv,
+            demonstrations=expert_trajs,
+            policy=sqil["policy_model"],
+            custom_logger=custom_logger,
+            rl_algo_class=sqil["rl_algo_class"],
+            rl_kwargs=sqil["rl_kwargs"],
+        )
+
+        sqil_trainer.train(
+            total_timesteps=int(sqil["total_timesteps"]),
+            **sqil["train_kwargs"],
+        )
+        sqil_trainer.save_policy(policy_path=osp.join(log_dir, "final.th"))
+
+        imit_stats = policy_evaluation.eval_policy(sqil_trainer.policy, venv)
+
+    stats = {"imit_stats": imit_stats}
+    expert_stats = _try_computing_expert_stats(expert_trajs)
+    if expert_stats is not None:
+        stats["expert_stats"] = expert_stats
+
+    return stats
+
+
 def main_console():
     observer_path = pathlib.Path.cwd() / "output" / "sacred" / "train_dagger"
```

Comment:
This script does a lot more than just dagger training, rename from `"train_dagger"` to `"train_imitation"`? (This is an existing problem not introduced in this PR but may as well fix.)

Reply:
> This script does a lot more than just dagger training

How so? It seems like everything it does (including creating a bc_trainer) is necessary for training dagger.

Reply:
Sorry, I thought you were commenting on the train_dagger function. This makes sense. Changed.